package.json

```diff
@@ -4,8 +4,10 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "npx ts-node src/main.ts",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "node ./dist/main.js",
+    "dev": "ts-node src/main.ts",
```

Comment:
Removed `npx` as you should be able to run this with just `ts-node`?

Reply:
I have no clue, if it works locally good enough for me.

Reply:
Had to look up what `npx` is, seems like it should be fine since its in dev deps.